lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -42,10 +42,16 @@ Stream<Patch> intlImporter(
 
   final insertInfo = _insertionLocationForPackageImport(
       intlUri, mainLibraryUnitResult.unit, mainLibraryUnitResult.lineInfo);
+
+  final hasPackageImport = insertInfo.hasPackageImports;
+  final hasRelativeImports = insertInfo.hasRelativeImports;
+
+  final importFormat = decideImportFormat(hasPackageImport, hasRelativeImports);
+
+  final importStatement = importFormat == 'package' ? insertInfo.leadingNewlines + "import '$intlUri'; " + insertInfo.trailingNewlines
+                                                    : insertInfo.leadingNewlines + "import '../../intl/${projectName}_intl.dart'; " + insertInfo.trailingNewlines;
```

Comment:
I don't think we know for certain that it will be two levels up. A file could be in lib/src, lib/src/project, lib/src/project/components, lib/src/project/components/report_component/redux, etc.

Reply:
Yes, files may be in different packages.